pkg/cmd/server/api/v1/swagger_doc.go

```diff
@@ -767,6 +767,6 @@ var map_VolumeConfig = map[string]string{
 	"localQuota": "LocalQuota contains options for controlling local volume quota on the node.",
 }
 
-func (VolumeConfig) SwaggerDoc() map[string]string {
+func (NodeVolumeConfig) SwaggerDoc() map[string]string {
```

Comment:
hack/update-generated-swagger-descriptions.sh should create one like this for MasterVolumeConfig